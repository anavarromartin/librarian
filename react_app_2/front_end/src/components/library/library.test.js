import React from 'react';
import renderer from 'react-test-renderer';
import Library from "./library"

describe('<Library />', () => {

    it('renders the component', () => {
        const component = renderer.create(<Library />)

        expect(component.toJSON()).toMatchSnapshot()
    })

})
