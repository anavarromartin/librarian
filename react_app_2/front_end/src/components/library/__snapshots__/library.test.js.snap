// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Library /> renders the component 1`] = `
<div
  className="library-container"
>
  <svg
    alt="logo"
    className="image-container"
  >
    home_page_img.svg
  </svg>
  <button
    className="teal-button"
    id="return-button"
    onClick={[Function]}
  >
    RETURN
  </button>
  <button
    className="white-button"
    id="borrow-button"
  >
    BORROW
  </button>
</div>
`;
